From 47d59f4e26559786fc3463a2427dcd0098b2efbb Mon Sep 17 00:00:00 2001
From: Marek Roszko <mark.roszko@gmail.com>
Date: Wed, 12 May 2021 23:30:19 -0400
Subject: [PATCH] Restore old wxObjectDataPtr copy constructor

---
 interface/wx/object.h | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/interface/wx/object.h b/interface/wx/object.h
index 513297a7ad..0fe02f9032 100644
--- a/interface/wx/object.h
+++ b/interface/wx/object.h
@@ -585,17 +585,11 @@ public:
     */
     wxObjectDataPtr(T* ptr = NULL);
 
-    //@{
     /**
         This copy constructor increases the count of the reference counted object to
         which @a tocopy points and then this class will point to, as well.
-
-        Using @a U different from @c T is only supported since wxWidgets 3.1.5.
     */
-    template <typename U>
-    wxObjectDataPtr(const wxObjectDataPtr<U>& tocopy);
     wxObjectDataPtr(const wxObjectDataPtr<T>& tocopy);
-    //@}
 
 
     /**
-- 
2.28.0.windows.1

