From 083b88fee7147f33cbb97d323f7d99832a0b309d Mon Sep 17 00:00:00 2001
From: Marek Roszko <mark.roszko@gmail.com>
Date: Wed, 12 May 2021 23:18:42 -0400
Subject: [PATCH] Add wx typedef hints

---
 etg/defs.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/etg/defs.py b/etg/defs.py
index 167e0726..4153e8a0 100644
--- a/etg/defs.py
+++ b/etg/defs.py
@@ -43,6 +43,7 @@ def run():
     module.find('wxInt64').type = 'long long'
     module.find('wxUint64').type = 'unsigned long long'
 
+
     # See src/wacky_ints.sip
     module.find('wxIntPtr').ignore()
     module.find('wxUIntPtr').ignore()
@@ -78,6 +79,9 @@ def run():
     module.insertItemAfter(td, etgtools.TypedefDef(type='SIP_SSIZE_T', name='ssize_t'))
     module.insertItemAfter(td, etgtools.TypedefDef(type='unsigned char', name='byte', pyInt=True))
     module.insertItemAfter(td, etgtools.TypedefDef(type='unsigned long', name='ulong'))
+    module.insertItemAfter(td, etgtools.TypedefDef(type='unsigned int', name='WXUINT'))
+    module.insertItemAfter(td, etgtools.TypedefDef(type='unsigned long', name='WXDWORD'))
+    module.insertItemAfter(td, etgtools.TypedefDef(type='unsigned short', name='WXWORD'))
 
 
 
-- 
2.28.0.windows.1

